benchmarks/rma_bw.c

```diff
@@ -125,6 +125,9 @@ int main(int argc, char **argv)
 		}
 	}
 
+	if (opts.rma_op == FT_RMA_WRITEDATA)
+		hints->mode |= FI_RX_CQ_DATA;
+
```

Comment:
Yes, we need this. The cq_data_size attribute would also have to be set (to 4 usually). msg_rma and rdm_rma tests need this change as well.